design/Proposal-to-enhance-schema-for-metal3.md

```diff
@@ -0,0 +1,228 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Proposal to enhance Schema for Metal3
```

Comment:
I can imagine that over time we will have a lot of schema enhancement requests. Could you give this proposal a more specific title (and rename the file to match)?

Reply:
Sure @dhellmann